Dockerfile

```diff
@@ -43,8 +43,8 @@ ENV DEBIAN_FRONTEND noninteractive
 ENV DEBCONF_NONINTERACTIVE_SEEN true
 
 ENV ANDROID_SDK_URL="https://dl.google.com/android/android-sdk_r24.4.1-linux.tgz" \
-    ANDROID_BUILD_TOOLS_VERSION=23.0.2 \
-    ANDROID_APIS="android-10,android-15,android-16,android-17,android-18,android-19,android-20,android-21,android-22,android-23,android-24" \
+    ANDROID_BUILD_TOOLS_VERSION=25.0.0 \
+    ANDROID_APIS="android-20,android-21,android-22,android-23,android-24" \
```

Comment:
@saikrishna321 I feel we should support API level 19 as well. Please refer to https://developer.android.com/about/dashboards/index.html